conf.py

```diff
@@ -0,0 +1,115 @@
+
```

Comment:
Can we put this file and the other `index.rst`s in a dedicated folder (e.g., `docs`) to avoid polluting the root of the repo?

Reply:
typically, yes, and that's the way I did it in other repos, the problem is that the docs site requires linking to documents under the directory that conf.py is located in. So the links to the README files will not work if we move conf.py into doc.

Reply:
This repo has a lot of files that need to be referenced outside of doc, such as the readme for each folder.

Reply:
I see (I found https://github.com/sphinx-doc/sphinx/issues/701).

At some point in the next few months, when working on the documentation, I'm still going to try to move it to a `doc` folder :slightly_smiling_face:. We don't have to reuse the per-package README files, we can document in one file (within `doc`) which packages exist and what they do.

Reply:
Yeah I agree, I think that makes sense